api/db-cdr/generate-cdr/make-bq-data-dictionary.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# This generates the big query data dictionary tables by importing data from
+# csv at gs://all-of-us-workbench-private-cloudsql/$BQ_DATASET/data_dictionary.
+
+set -ex
+
+export BQ_PROJECT=$1  # project
+export BQ_DATASET=$2  # dataset
+export DRY_RUN=$3     # dry run
+
+archiveList=$(gsutil ls gs://all-of-us-workbench-private-cloudsql/$BQ_DATASET/data_dictionary/archive/*.csv) > /dev/null || true;
+dataDictionaryList=$(gsutil ls gs://all-of-us-workbench-private-cloudsql/$BQ_DATASET/data_dictionary/*.csv) > /dev/null || true;
+
+if [ "$DRY_RUN" == true ]
+then
+  echo "Check if data dictionary file exists"
+  if [ -z "$dataDictionaryList" ] && [ -z "$archiveList" ]
+  then
+    echo "Error: Data Dictionary file is missing!"
+    exit 1;
+  fi
+  exit 0
+fi
+
+# Throw error if there is no data_dictionary file for CDR release
+echo "Check if data dictionary file exists"
+if [ -z "$dataDictionaryList" ] && [ -z "$archiveList" ]
+then
+  echo "Error: Data Dictionary file is missing!"
+  exit 1;
+fi
+
+tableCreated=false;
+
+for filename in $dataDictionaryList; do
```

Comment:
If we only have should have 1 csv file per process then this can just be an if instead of a iteration

Reply:
Done